### PR TITLE
chore: release 2.14.0

### PR DESCRIPTION
**Pull request by dbrasseur-aneo**

### CODE REVIEW

File: armonik-versions.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 infra=0.0.2
-core=0.13.2
-worker=0.11.1
+core=0.14.2
```
Comment (lzianekhodja) [inline armonik-versions.txt:2]: Is version 0.14.0 or 0.14.2 for Core? (see changlog)

Comment (aneojgurhem) [reply armonik-versions.txt:2]: 0.14.2

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,7 +32,7 @@ Changed
 * Upgrade Core version from `0.13.1` to `0.13.2`.
 
 Fixed
-
+-
```
Comment (aneojgurhem) [inline CHANGELOG.md:35]: Is this needed ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,11 +2,26 @@
...
 
+## [v2.14.0](https://github.com/aneoconsulting/armonik/tree/v2.14.0) (2023-07-05)
+
 Changed
 -
 
 * Move Terraform modules of different resources deployment to GitHub repository [ArmoniK.Infra](https://github.com/aneoconsulting/ArmoniK.Infra).
 * Use ArmoniK Infrastructure version `0.0.2`
+* Upgrade ArmoniK.Extension.Csharp version from `0.11.1` to `0.12.0`
+* Upgrade Core version from `0.13.2` to `0.14.0`
```
Comment (lzianekhodja) [inline CHANGELOG.md:13]: Replace 0.14.0 by 0.14.2

File: armonik-versions.txt (hunk 1 above)
Comment (lzianekhodja) [reply armonik-versions.txt:2]: OK we must update changelog

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,11 +2,26 @@
...
+Added
+-
+
+* Grpc channel health check when recycling in Extension.Csharp
+
+Fixed
+-
+
+* Grpc channel keep alive configuration in Api
+* Retry using bad channel in Extension.Csharp
```
Comment (ngruelaneo) [inline CHANGELOG.md:24]: Il faut mettre en avant le TTl sur redis avec la variable: TtlTimeSpan

Comment (ngruelaneo) [reply CHANGELOG.md:24]: https://github.com/aneoconsulting/ArmoniK.Core/pull/419

Comment (aneojgurhem) [reply CHANGELOG.md:24]: `Redis__TtlTimeSpan`

File: CHANGELOG.md (hunk 2 above)
Comment (dbrasseur-aneo) [reply CHANGELOG.md:35]: It wasn't formatted properly

**Review by aneojgurhem — APPROVED** [review]

**Review by lzianekhodja — DISMISSED** [review]

**Comment by esoubiran-aneo** [comment]
GUI is broken since we broke the sort field in API.

**Comment by esoubiran-aneo** [comment]
I'm currently fixing it. Will update this PR when it will be released.

**Review by esoubiran-aneo — DISMISSED** [review]
GUI is broken. I'm working on it.

**Comment by esoubiran-aneo** [comment]
@dbrasseur-aneo commit [73f47f3](https://github.com/aneoconsulting/ArmoniK/pull/1163/commits/73f47f3ad9de455298bcffd47bba3bc27b6cfee4) seems to be broken when metrics export install because image is no found.

**Comment by dbrasseur-aneo** [comment]
> @dbrasseur-aneo commit [73f47f3](https://github.com/aneoconsulting/ArmoniK/pull/1163/commits/73f47f3ad9de455298bcffd47bba3bc27b6cfee4) seems to be broken when metrics export install because image is no found.

That's expected, Core hasn't been released yet. I'll rerun the ci when we do the release

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,11 +2,44 @@
...
+* Worker abort propagation
+* Reduced database load
+  * Removed unnecessary indices
+  * Optimized requests
```
Comment (dbrasseur-aneo) [inline CHANGELOG.md:43]:
```suggestion

## [v2.13.3](https://github.com/aneoconsulting/armonik/tree/v2.13.2) (2023-07-05)

Changed
-

* Use Armonik.Extension.Csharp version `0.11.2`

Fixed


* Bad properties override

* 

```

**Review by ngruelaneo — APPROVED** [review]

**Pull request merged**